reconcile/utils/gql.py

```diff
@@ -30,6 +33,10 @@
 }
 """
 
+requests_logger.setLevel(logging.WARNING)
+
+_lock = Lock()
```

Comment:
Isn't the issue that the `Client` blocks on `execute` and can only run one at a time? If so, shouldn't the `Lock` really be part of the class? I'd also make it clear what the purpose of the lock is.

Reply:
Removed the use of lock.